front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx

```diff
@@ -34,10 +35,11 @@ export default function UserInfoToolbar() {
     dispatch({type: 'SET_MODAL', payload: false});
     clearRecipe();
   };
-
+  // app, recipe page, user
```

Comment:
I think these might just be notes about where the router stuff is located? If you're not usin 'em anymore, I recommend deleting!